internal/oci/runtime_oci.go

```diff
@@ -262,7 +262,7 @@ func (r *runtimeOCI) ExecContainer(ctx context.Context, c *Container, cmd []stri
 	}
 	defer os.RemoveAll(processFile)
 
-	execCmd := r.constructExecCommand(ctx, c, processFile, "")
+	execCmd := r.constructExecCommand(ctx, c, processFile, "", tty)
```

Comment:
We are missing this in release-1.21.

Reply:
oh yes this is not actually needed anymore

Reply:
updated